web/looking-good/src/utils/auth-service.js

```diff
@@ -142,4 +142,11 @@ export default class AuthService {
       method: 'GET',
     })
   }
+
```

Comment:
@ben-beers you can call this function to delete the user's profile, nice and easy.

Reply:
Thanks! This will be the rest of my evenin + tomorrow. :-)